public/javascripts/SVValidate/src/mission/Mission.js

```diff
@@ -11,6 +11,7 @@ function Mission(params) {
         disagreeCount: 0,
         missionId: undefined,
         missionType: undefined,
+        missionsCompleted: undefined,
```

Comment:
Do you still need the parts you added in this file that involve completed missions?

Reply:
missionsCompleted is legacy code use in other functions (such as show(mission) in ModalMissionComplete).

Reply:
Sure, but you added these lines

Reply:
looked over again, and that value does not need to be part of the mission object. Not sure what I was thinking when writing that code